cmd/mt-whisper-importer-reader/main.go

```diff
@@ -205,7 +205,7 @@ func getMetricName(file string) string {
 		file = file[1:]
 	}
 
-	return *namePrefix + strings.Replace(strings.TrimSuffix(file, ".wsp"), "/", ".", -1)
+	return *namePrefix + strings.Replace(strings.TrimSuffix(strings.TrimPrefix(file, *whisperDirectory), ".wsp"), "/", ".", -1)
```

Comment:
that's great. 
is it possible that you'll need to take care of a possible leading `/` that's omitted in `whisperDirectory`?
like f.e. if the `whisperDirectory` is `/opt/storage` and the `file is `/opt/storage/mypath/abc.wsp` then we'll want `mypath/abc.wsp` and not `/mypath/abc.wsp` right?

Reply:
Yea, that's definitely an issue,  since there could be a `.` prefixing the name
https://play.golang.org/p/lLZtHioeJO